webpack.config.js

```diff
@@ -125,6 +125,10 @@ module.exports = {
           from: 'node_modules/leaflet/dist/images',
           to: 'css/images',
         },
+        {
+          from: 'node_modules/prismjs/components/*.min.js',
+          to: 'js/prismjs/components/[name].js',
```

Comment:
Prism language packs are copies to this build folder. Depending on the code snippets in markdown, it will be fetched.